### DISCUSSION TITLE
Suppress test case output while initialising clock?

### DISCUSSION THREAD

**Discussion by binaryO11O** [hneemann/Digital#1126] [Q&A] [answered]
Just getting started with Digital today after using Logisim for many years. Very impressed!!

I'm writing a case that needs to repeat through clock cycles before doing the actual test.

```
1     C  C0 HS VS
2  
3  while (!VS)
4     C  x  x  x
5  end while
6
7     C  x  0  1
```
in this scenario, it takes many thousands of clock cycles until VS switches off.

The Test Result however only shows me the first 1024 rows (all L4) and then says **passed (too many entries)**

Is there anyway that I can suppress L4 and only display the result for L7?

**Comment by hneemann** [comment, marked as answer]
It is not possible to suppress some lines. But if an error occurs, the line will be displayed in any case. So if the X is replaced by a 1 or 0, the test is either green (the signals match) and thus not visible or red, then it is also displayed.
The purpose of the tests is not to have to check yourself if the signals are correct, but to let the computer do it.
And then it is enough if the incorrect outputs are displayed.

**Reply by binaryO11O** [reply]
I see!!! I didn't realise that it would still display the failed case after maxxing out on the number of passed tests.

Thank you so much for the fast response and for a magnificent educational tool!

**Reply by hneemann** [reply]
You are welcome! :smile:
